### PR TITLE
feat: Easy extraction of qualified volunteer list for nomcom chair

### LINKED ISSUE #4732: Generating volunteer list
### Description

It would be nice to have an option to generate the volunteer list that just output text/plain with numbering.
```
1. Rich Salz
2. Robert Sparks
3. ...
```

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by rjsparks**
Fixes #4732.

Replaces #5412

### CODE REVIEW

**Comment by rjsparks** [comment]
@richsalz Please review

**Review by jennifer-richards — APPROVED** [review]

**Review by richsalz — APPROVED** [review]
One comment, which can be ignored.

File: ietf/templates/nomcom/qualified_volunteer_list_for_announcement.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{% load ietf_filters %}{% autoescape off %}Name                     Affiliation
+
+{% for volunteer in volunteers %}{{ volunteer.person.plain_name|ljust:"25" }}{{ volunteer.affiliation }}
```
Comment (richsalz) [inline ietf/templates/nomcom/qualified_volunteer_list_for_announcement.txt:3]: I wonder if you can explicitly put "none supplied" or something if affiliation is empty?

File: ietf/nomcom/views.py [hunk 2]
```diff
@@ -1333,3 +1333,12 @@ def private_volunteers_csv(request, year, public=False):
         writer.writerow([v.person.last_name(), v.person.first_name(), v.person.ascii_name(), v.affiliation, v.person.email(), v.qualifications, v.eligible])
     return response
 
+@role_required("Nomcom Chair", "Nomcom Advisor", "Secretariat")
+def qualified_volunteer_list_for_announcement(request, year, public=False):
+    _, volunteers = extract_volunteers(year)
```
Comment (richsalz) [inline ietf/nomcom/views.py:1338]: I detect the essence of `go` or `rust` :)

**Review by richsalz — APPROVED** [review]

**Pull request merged**